packages/passport/sdk/src/Passport.ts

```diff
@@ -299,4 +297,26 @@ export class Passport {
     const getUserInfoResult = await this.multiRollupApiClients.passportProfileApi.getUserInfo({ headers });
     return getUserInfoResult.data.linked_addresses;
   }
+
+  public async linkExternalWallet(params: LinkWalletParams): Promise<LinkedWallet> {
+    track('passport', 'linkWallet', { type: params.type });
+    const user = await this.authManager.getUser();
+    if (!user) {
+      throw new PassportError('User is not logged in', PassportErrorType.NOT_LOGGED_IN_ERROR);
+    }
+    const headers = { Authorization: `Bearer ${user.accessToken}` };
+    const linkWalletV2Request = {
+      type: params.type,
+      wallet_address: params.walletAddress,
+      signature: params.signature,
+      nonce: params.nonce,
+    };
+    const linkWalletV2Result = await this.multiRollupApiClients
+      .passportProfileApi
+      .linkWalletV2(
+        { linkWalletV2Request },
+        { headers },
+      );
+    return { ...linkWalletV2Result.data };
```

Comment:
Should we check if the user is registered with zkEvm or IMX before allowing the request to be sent?
And how are errors handled here? What type of error is thrown from this function if the request fails? If it's an Axios error, then we probably want to wrap it and handle accordingly.